pkg/asset/manifests/cluster_k8s_io.go

```diff
@@ -53,7 +53,7 @@ func (c *ClusterK8sIO) Generate(dependencies asset.Parents) error {
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      installconfig.Config.ObjectMeta.Name,
-			Namespace: "openshift-cluster-api",
```

Comment:
If you are using this object on the actuator, you probably need to update the APIVersion version here `APIVersion: "cluster.k8s.io/v1alpha1"` to be consistent with what the actuator is taking

Reply:
should be fine. I can prolly change this in a follow-up patch